playbooks/configure_tmux.yml

```diff
@@ -1,8 +1,12 @@
 ---
 - hosts: hosts
   user: ubuntu
+  vars:
+    tmux_config: |
+      set-option -g history-limit 40000
+      set-option -g mode-keys vi
```

Comment:
Why?

Reply:
@johnduarte Copy mode is hard to use without the vi mode keys. Makes it a lot simpler: https://tmuxcheatsheet.com/?q=copy&hPP=100&idx=tmux_cheats&p=0&is_v=1